table/sst_file_writer.cc

```diff
@@ -231,6 +232,8 @@ Status SstFileWriter::Open(const std::string& file_path) {
   }
 
   sst_file->SetIOPriority(r->io_priority);
+  r->kv_validator = std::unique_ptr<OutputValidator>(
+      new OutputValidator(r->internal_comparator, /* enable_order_check */ true, /* enable_hash */true));
```

Comment:
Is it over 80-char?

Reply:
Not sure, can't tell from VS Code unless I manually count. But the formatter should have caught it, no?